src/common/symop.jl

```diff
@@ -0,0 +1,34 @@
+# Represents a symmetry (S,τ)
+struct SymOp
+    S::Mat3{Int}
+    τ::Vec3{Float64}
+    function SymOp(S, τ)
+        τ = τ .- floor.(τ)
+        @assert all(0 .≤ τ .< 1)
+        new(S, τ)
+    end
+    SymOp(Sτ::Tuple) = SymOp(Sτ...) # compatibility with old stuff, doesn't hurt
+end
+
+Base.:(==)(op1::SymOp, op2::SymOp) = op1.S == op2.S && isapprox(op1.τ, op2.τ; atol=1e-8)
```

Comment:
Why `1e-8`? Our default symmetry tolerance is `1e-5`.
Maybe better to put that constant somewhere? Or make it depend on the type of `τ`.

Reply:
OK I overhauled it, tell me what you think

Reply:
Yes that's better. Equality should really be equality.